### PR TITLE
Implemented data observatory measure analysis

### PR DESCRIPTION
**Pull request by dgaubert**
cc @rochoa

### CODE REVIEW

File: lib/node/nodes/data-observatory-measure.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+dot.templateSettings.strip = false;
+
+var Node = require('../node');
+
+var TYPE = 'data-observatory-measure';
+var PARAMS = {
+    source: Node.PARAM.NODE(Node.GEOMETRY.POLYGON),
+    new_column_name: Node.PARAM.STRING,
+    segment_name: Node.PARAM.STRING,
+    percent: Node.PARAM.BOOLEAN,
```
Comment (rochoa) [inline lib/node/nodes/data-observatory-measure.js]: Should we make this NULLABLE?

File: lib/node/nodes/data-observatory-measure.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+var dot = require('dot');
+dot.templateSettings.strip = false;
+
+var Node = require('../node');
+
+var TYPE = 'data-observatory-measure';
+var PARAMS = {
+    source: Node.PARAM.NODE(Node.GEOMETRY.POLYGON),
```
Comment (rochoa) [inline lib/node/nodes/data-observatory-measure.js]: I though it was possible to apply it over points, did I miss anything?

Comment (dgaubert) [inline lib/node/nodes/data-observatory-measure.js]: I've just followed the provided description `Calculate Data Observatory measure within a Polygon`. But there isn't any  constraint. ;)

Comment (dgaubert) [inline lib/node/nodes/data-observatory-measure.js]: I've just update it. I think there is no reason to keep the constraint.

File: lib/node/nodes/data-observatory-measure.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+var dot = require('dot');
+dot.templateSettings.strip = false;
+
+var Node = require('../node');
+
+var TYPE = 'data-observatory-measure';
+var PARAMS = {
+    source: Node.PARAM.NODE(Node.GEOMETRY.ANY),
```
Comment (rochoa) [inline lib/node/nodes/data-observatory-measure.js]: Hehe, does it support lines?

Comment (rochoa) [inline lib/node/nodes/data-observatory-measure.js]:
```
Node.PARAM.NODE(Node.GEOMETRY.POINT, Node.GEOMETRY.LINESTRING)
```

Comment (dgaubert) [inline lib/node/nodes/data-observatory-measure.js]: DO returns null for lines.

**Comment by rochoa** [comment]
🎉 !!! Thanks a lot.

**Pull request merged**